packages/front-end/components/Features/FeatureUsageGraph.tsx

```diff
@@ -0,0 +1,163 @@
+import { Bar } from "@visx/shape";
+import { scaleBand, scaleLinear } from "@visx/scale";
+import { ParentSizeModern } from "@visx/responsive";
+import { Group } from "@visx/group";
+import React, { createContext, ReactNode, useContext, useEffect } from "react";
+import {
+  FeatureUsageData,
+  FeatureUsageTimeSeries,
+} from "back-end/types/feature";
+import { FeatureUsageLookback } from "back-end/src/types/Integration";
+import useApi from "@/hooks/useApi";
+import { useLocalStorage } from "@/hooks/useLocalStorage";
+import { growthbook } from "@/services/utils";
+
+const formatter = Intl.NumberFormat("en-US", {
+  notation: "compact",
+  maximumFractionDigits: 1,
+});
+
+const featureUsageContext = createContext<{
+  lookback: FeatureUsageLookback;
+  setLookback: (lookback: FeatureUsageLookback) => void;
+  featureUsage: FeatureUsageData | undefined;
+  mutateFeatureUsage: () => void;
+}>({
+  lookback: "15minute",
+  setLookback: () => {},
+  featureUsage: undefined,
+  mutateFeatureUsage: () => {},
+});
+
+export function FeatureUsageProvider({
+  featureId,
+  children,
+}: {
+  featureId: string;
+  children: ReactNode;
+}) {
+  const [lookback, setLookback] = useLocalStorage<FeatureUsageLookback>(
+    "featureUsageLookback",
+    "15minute"
+  );
+
+  const useFeatureUsage = growthbook.isOn("feature-usage");
+
+  const { data: featureUsage, mutate: mutateFeatureUsage } = useApi<{
+    usage: FeatureUsageData;
+  }>(`/feature/${featureId}/usage?lookback=${lookback}`, {
+    shouldRun: () => useFeatureUsage,
+  });
+
+  useEffect(() => {
+    if (!featureUsage) return;
+    if (lookback !== "15minute") return;
+    const timer = setInterval(
+      () => {
+        mutateFeatureUsage();
+      },
+      // Update slower when there's no data yet
+      featureUsage.usage?.overall?.total ? 2000 : 5000
```

Comment:
Since we're hitting ClickHouse directly without a cache layer, I'm a little worried about auto-refreshing right now.  Maybe we can add another feature flag with the interval time so we can control this remotely.  If we notice high load on ClickHouse, we can set the interval higher or set it to `0` to disable auto-refresh entirely.

That would also let us have a fast refresh on our demo account so we can show off the graphs better.